Tools/WinMLRunner/CommandLineArgs.h

```diff
@@ -82,6 +86,7 @@ class CommandLineArgs
     bool m_silent = false;
     bool m_autoScale = false;
     BitmapInterpolationMode m_autoScaleInterpMode = BitmapInterpolationMode::Cubic;
+    bool m_inputImagePreProcess = false;
```

Comment:
nit: i would prefer if the below members are part of a struct, and stored in an std::optional<>. That way you dont need additional tracking boolean flags to see if its been set.